readme.md

```diff
@@ -146,6 +146,15 @@ Similar to [`childProcess.kill()`](https://nodejs.org/api/child_process.html#chi
 
 Stream combining/interleaving [`stdout`](https://nodejs.org/api/child_process.html#child_process_subprocess_stdout) and [`stderr`](https://nodejs.org/api/child_process.html#child_process_subprocess_stderr).
 
+### execa.fork(file, [arguments], [options])
+
+Run a file through a forked process. The default sub-process is the current (node).
+
+It introduce a communication channel (IPC) that allows messages to be passed back and forth between the parent and child.<br>
+Note that the `shell` option will be ignored.
+
+Returns the same object as [execa](#execafile-arguments-options).
```

Comment:
I find the explanation a little confusing:
  - For example this sentence: `The default sub-process is the current (node).`
  - the `IPC` channel is available to `execa()` as well, not just by default.

What about the following?

```
Same as `execa('node', [file, ...arguments], options)` except (like [`child_process#fork()`](https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options)):
  - the `execPath`, `execArgv` and `silent` options can be used
  - the [`shell`](#shell) option cannot be used 
  - the [`stdio`](#stdio)-related options defaults to `inherit` instead of `pipe`
```

Reply:
It was copied from the [node API documentation](https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options), but your version looks good too.

Reply:
Just for clarification, only the IPC sentence was.